contract/src/airdrop/prepareAirdropKit.js

```diff
@@ -0,0 +1,341 @@
+import { M } from '@endo/patterns';
+import {
+  makeScalarBigSetStore,
+  prepareExoClassKit,
+  provide,
+} from '@agoric/vat-data';
+import {
+  IssuerShape,
+  PaymentShape,
+  BrandShape,
+  PurseShape,
+  AmountMath,
+  DepositFacetShape,
+} from '@agoric/ertp';
+import {
+  atomicRearrange,
+  fromOnly,
+  provideAll,
+  toOnly,
+} from '@agoric/zoe/src/contractSupport/index.js';
+import { E, Far } from '@endo/far';
+import { makePromiseKit } from '@endo/promise-kit';
+import { AIRDROP_ADMIN_MESSAGES, CLAIM_MESSAGES } from './helpers/messages.js';
+import '../../types.js';
+import { makeCancelTokenMaker } from './helpers/time.js';
+
+const head = ([x, ...xs]) => x;
+
+const makeTracer = label => value => {
+  console.log(label.toUpperCase(), '::::', value);
+  return value;
+};
+
+const tracer = makeTracer('Airdrop airdropCampaign');
+/** @type {airdropCampaignMeta} */
+export const meta = {
+  upgradability: 'canUpgrade',
+  terms: {
+    proofHolderFacet: M.remotable('proofHolder Powers'),
+  },
+  privateArgsShape: M.splitRecord({
+    powers: {
+      timerService: M.eref(M.remotable('TimerService')),
+    },
+  }),
+};
```

Comment:
```suggestion
};
// for compatibility with agoric-upgrade-13 etc.
export const customTermsShape = meta.customTermsShape;
export const privateArgsShape = meta.privateArgsShape;
```

Reply:
oh... and this has to exported from the main contract module in order for Zoe to notice it.